src/operations/result/ResultOperations.ts

```diff
@@ -24,6 +24,7 @@ export class ResultOperations<TOptions extends OperationOptions> extends Operati
   public async get(params: ParamsToGetResult): Promise<ResponseFromGetResult> {
     const { accessToken, resultId } = params;
     const response = await this.sendGetRequest<ResponseFromGetResult>({
+      // eslint-disable-next-line  @typescript-eslint/no-non-null-assertion
```

Comment:
not a fan of this rule; how can you guarantee the cb is defined?

Reply:
Good point. I'll make a change to test for this.